docs/development/localization.md

```diff
@@ -6,6 +6,36 @@ in basedpyright we want to fix this but we need your help! if you are fluent in
 
 Here are some guidelines for contributors who would like to help improve the translations in basedpyright.
 
+## Tools for localization
+
+A TUI tool `build/localization_helper.py` script is provided to help you with the localization process. It can be used to:
+
+-   Check every message in comparison with the corresponding English message.
+-   Compare message keys with the English version and find out which messages are missing and which ones are redundant.
+
+### Usage
+
+Run the script:
+
+```shell
+# use uv
+uv run build/localization_helper.py
+# or pdm
+pdm run build/localization_helper.py
+# or run the script directly IF YOU CAN ASSURE YOU ARE ALREADY IN THE VIRTUAL ENVIRONMENT OF THE PROJECT
+python build/localization_helper.py
```

Comment:
this project doesn't use pdm anymore since we've switched to uv. i now try to keep all project scripts centralized in [the top-level `package.json`](https://github.com/DetachHead/basedpyright/blob/a4e7c9455806140ea095301e9c5df3b529e254dd/package.json#L28), so we should probably just reference that instead.

ideally the script would be defined in `pyproject.toml` like it was when we were using pdm, but uv does not support this yet (see https://github.com/astral-sh/uv/issues/5903). however this means running it from outside the venv is annoying because it has to go through 3 different tools:

```shell
./pw uv run npm run localization-helper
```

so maybe the instructions should instead be written like this:

````suggestion
```shell
# use uv
./pw uv run build/localization_helper.py
# or from inside the venv
npm run localization-helper
````

the reason i have it set up this way is so that all these tools are installed and managed locally in the project, and the developer never has to globally install any tools like uv, pdm or node